charts/llmisvc-resources/templates/gateway-inference-extension.yaml

```diff
@@ -0,0 +1,540 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.16.1
+  name: inferencemodels.inference.networking.x-k8s.io
```

Comment:
Do we want to install GIE as part of kserve?

Reply:
I think generally we want to have some control of the versioning of GIE (as Kserve is the primary controller of GIE and EPP), however, we could have a bring your own option